packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -179,115 +179,57 @@ export class LitNodeClientNodeJs
   createCapacityDelegationAuthSig = async (
     params: CapacityCreditsReq
   ): Promise<CapacityCreditsRes> => {
-    const {
-      dAppOwnerWallet,
-      capacityTokenId,
-      delegateeAddresses,
-      uses,
-      domain,
-      expiration,
-      statement,
-    } = params;
+    // Useful log for debugging
+    if (!params.delegateeAddresses || params.delegateeAddresses.length === 0) {
```

Comment:
How about making passing `delegateeAddresses` mandatory & when provided empty array that means anyone can use it?

Reply:
What do you think @Ansonhkg?

Reply:
Providing an empty array means no one can use it though, I think we should not allow an empty array instead